lib/que/job.rb

```diff
@@ -101,6 +120,47 @@ def enqueue(*args)
       end
       ruby2_keywords(:enqueue) if respond_to?(:ruby2_keywords, true)
 
+      def bulk_enqueue(args_and_kwargs_array, job_options: {})
```

Comment:
It'd probably be nice to share as much code as we can between this and the original `.enqueue` method

Reply:
Should be easy to share the job options construction, args are treated pretty differently so we’ll have to keep that on the respective methods.